i18n/ko.json

```diff
@@ -268,110 +280,112 @@
     "total": "합계",
-    "videoFolder": "여기에있는 비디오 :",
-    "videoLengths": "비디오 길이"
+    "videoFolder": "동영상 위치:",
+    "videoLengths": "동영상 길이"
   },
   "SYSTEM": {
     "cancel": "취소",
-    "continueAnyway": "어쨌든 계속",
-    "directory": "디렉토리",
-    "doesNotExist": "없습니다.",
-    "hubAlreadyExists": "같은 이름으로 허브가 존재합니다.",
-    "noSuchFileFound": "해당파일을 찾을 수 없습니다. :",
-    "pleaseChangeName": "허브이름을 변경하십시오.",
-    "selectDefaultPlayer": "선호하는 비디오 플레이어를 선택하십시오.",
-    "selectPreviousHub": "이전에 저장된 허브파일을 선택하십시오.",
-    "selectRootFolder": "소스비디오 폴더 선택",
-    "videoFolderNotFound": "소스비디오 폴더를 찾을 수 없습니다. :"
+    "continueAnyway": "무시하고 계속",
+    "directory": "디렉터리",
+    "doesNotExist": "존재하지 않습니다.",
+    "HubAlreadyExists": "같은 이름의 Hub가 있음",
+    "noSuchFileFound": "해당 파일을 찾을 수 없음:",
+    "pleaseChangeName": "위 Hub 이름을 변경하십시오.",
+    "selectDefaultPlayer": "선호하는 동영상 플레이어를 선택하세요.",
+    "selectPreviousHub": "이전에 저장한 동영상 Hub 파일을 선택하세요.",
+    "selectRootFolder": "원본 동영상 폴더 선택",
+    "videoFolderNotFound": "원본 동영상 폴더를 찾을 수 없음:"
   },
   "TAGS": {
-    "add": "추가",
-    "addSuccess": "성공적으로 추가되었습니다.",
-    "addTag": "태그 추가",
-    "alreadyExists": "태그가 이미 존재합니다.",
-    "closeTagTray": "태그트레이 닫기",
-    "delete": "삭제 모드",
-    "filterList": "필터목록",
-    "filterTags": "필터태그",
+    "add": "Add",
+    "addSuccess": "추가됨",
+    "addTag": "Add tag",
+    "alreadyExists": "태그가 이미 있음",
+    "any": "any",
+    "closeTagTray": "태그 트레이 닫기",
+    "delete": "delete mode",
+    "filterList": "Containing",
+    "filterTags": "필터 태그",
     "hidingFrequency": "빈도 숨기기",
-    "presentTags": "이 허브의 태그",
-    "showingFrequency": "게재 빈도",
-    "showingMostSimilar": "와 비슷한 모습을 보이는 :",
-    "sortAlphabetical": "알파벳순으로 정렬",
-    "sortByFrequency": "빈도별로 정렬",
-    "starRating": "별점 매기기",
+    "minimum": "minimum",
+    "presentTags": "이 Hub의 태그",
+    "showingFrequency": "빈도 보이기",
+    "showingMostSimilar": "비슷한 영상:",
+    "sortAlphabetical": "이름순 정렬",
+    "sortByFrequency": "빈도로 정렬",
+    "starRating": "별점 등급",
     "timesPlayed": "재생 횟수"
   },
   "WIZARD": {
     "change": "변경",
-    "clearRecent": "모든 기록을 지웁니다. ",
-    "clipSnippets": "단편",
+    "clearRecent": "최근의 모든 기록 지우기",
+    "clipSnippets": "조각으로",
     "closeButton": "마법사 닫기",
-    "createVideoHub": "비디오허브 만들기",
-    "demoVersion": "비디오파일 50개로 제한된 데모버전",
-    "eachSnippet": "각 조각들",
-    "expected": "예상 :",
-    "extractClips": "로 클립 미리보기 생성",
-    "found": "찾음",
-    "hubDirectory": "허브 디렉토리",
-    "imported": "삽입",
-    "importingFrom": "에서 가져오기 :",
-    "importNew": "새로운 허브 만들기",
-    "loadFromFile": "파일에서 불러오기",
-    "minutes": "분",
-    "nameYourHub": "허브이름",
-    "oneScreenshotEvery": "스크린샷 1개마다",
-    "openPrevious": "이전허브 열기",
-    "openPreviouslySaved": "이전에 저장된 비디오 허브 열기",
-    "recentlyOpened": "최근에 열었던 허브 :",
+    "createVideoHub": "동영상 Hub 만들기",
+    "demoVersion": "데모 버전은 50개 동영상으로 제한",
+    "eachSnippet": "각각의 조각들을",
+    "expected": "Expected:",
+    "extractClips": "클립 미리 보기 생성",
+    "found": "found",
```

Comment:
Should these be in English? (this and `expected`, `minimum`, `any`, etc above) ?

Reply:
On my level, it's hard to explain. So even if you don't have enough explanation 
,please understand.
I could see this sentence in the auto tag tray.
[containing any 4 minimum] the translation of this sentence 
In Korean, it is natural to be in the order of [minimum 4 any containing].

[339 line "filterList": "Containing"] requires to enter ['minimum' instead of {Containing}].
[342 line "minimum": "Minimum"] requires to enter ['Containing' instead of {'minimum'}].
{any, 4} -> {4, any} must be.  Unnatural `5stars`
So I couldn't change it.

Sometimes it's not natural if a sentence consists of multiple lines.
The wizard's case is similar. (2,3,4,5) Unnatural `2stars`

It is natural that the sentence of [select video, hub directory] 
should be in the order of [video, hub directory select].
So I used [select (video,hub directory).

Wizard again - import new hub - 4.2
{369 line "oneScreenshotEvery": "1 screenshot every",} -> {"oneEveryScreenshot": "","}, turn it into a blank
{367 line "minutes": "minutes",} -> {"minutes": "minutes 1 screenshot every",} can I change it to translate it?
Can I change the other parts as above?

I wanted to change it naturally, but it was because of my low level.

I should have done it better, but I couldn't.
I'm sorry I've uploaded in a hurry 
I'll check back and update

Reply:
I'll trust whatever you decide on -- I'm aware languages structure things differently, and words like 'or' and 'and' may be widely understood too (you would know better).

Thank you for your work 🙇 

No rush 👍